src/main/java/lukemccon/airdrop/controllers/PackageController.java

```diff
@@ -28,4 +40,64 @@ public static boolean onCommand(CommandSender sender, String[] args) {
 		return true;
 	}
 
+	public static boolean deletePackageCommand(CommandSender sender, String[] args) {
+		if (args.length != 3 ) {
+			ChatHandler.sendErrorMessage(sender, "Need to specify a package name to delete");
+		}
+
+		String packageName = args[2];
+		try {
+			PackageManager.deletePackage(packageName);
+			ChatHandler.sendMessage(sender, ChatColor.AQUA + packageName + ChatColor.BLUE + " was successfully deleted");
+		} catch (PackageNotFoundException e) {
+			ChatHandler.sendErrorMessage(sender, "Unable to delete package " + ChatColor.DARK_RED + packageName + ChatColor.RED + " not found");
```

Comment:
Should probably have some punctuation. Either ```Unable to delete package: <package name> not found``` or a period after unable to delete package.